ext/web/06_streams.js

```diff
@@ -218,15 +219,8 @@
    * @returns {ArrayBufferLike}
    */
   function transferArrayBuffer(O) {
-    assert(!isDetachedBuffer(O));
-    const transferredIshVersion = O.slice(0);
-    ObjectDefineProperty(O, "byteLength", {
-      get() {
-        return 0;
-      },
-    });
-    O[isFakeDetached] = true;
-    return transferredIshVersion;
+    const v = ops.op_transfer_arraybuffer(O);
+    v[isFakeDetached] = true;
```

Comment:
```suggestion
    O[isFakeDetached] = true;
    return v;
```

Reply:
https://github.com/denoland/deno/blob/2300a70f4148106385839f2401f4e29adb9c97df/ext/web/06_streams.js#L196-L198

And we can change that method to:

```js
 function isDetachedBuffer(O) { 
	   return O.byteLength === 0;
} 
```

Reply:
I'm keeping `isFakeDetached` to not touch a lot of code.

Reply:
> I'm keeping `isFakeDetached` to not touch a lot of code.

Could you fix this properly rather than half fixing it just enough so that WPT passes? 😉

Reply:
@lucacasonato https://github.com/denoland/deno/pull/16307 @marcosc90 opened a PR to do that as a follow up to this one.